rotkehlchen/types.py

```diff
@@ -507,3 +494,16 @@ class EnsMapping(NamedTuple):
 class CostBasisMethod(SerializableEnumMixin):
     FIFO = 1
     LIFO = 2
+
+
+class DataSource(SerializableEnumMixin):
```

Comment:
I agree. `Exchange` perhaps is not the best word since csv is not something that can come only from exchanges, but `DataImportSource` sounds better indeed. How do you think, will it will be sufficient?